src/lib/components/identity/identity-manager.svelte

```diff
@@ -83,9 +92,16 @@
     } as TableData
 
     onMount(async () => {
+        // Get the current filter state from store and set value accordingly
+        const { creatorFilterState } = get(identityFilterOptions)
```

Comment:
```suggestion
        const { creatorFilterState } = get(identityFilterOptions) ?? {}
```
just to make sure you can deconstruct it

Reply:
Added